### PR TITLE
Landing page design

### PR DESCRIPTION
**Pull request by aysam199**
Making the page more responsive, adds on enter click search, fixes naming for folders and files of selected course page

### CODE REVIEW

File: client/src/components/Header/Header.js [hunk 1]
```diff
@@ -11,6 +10,10 @@ function Header() {
     }
     return `/search/${searchTerm}`;
   };
+  const [searchEnter, setsearchEnter] = useState(false);
+    // useEffect(() => {
+      
+    // }, [searchTerm]);
```
Comment (Karyum) [inline client/src/components/Header/Header.js]: no need for this commented code, correct ?

File: client/src/components/Header/Header.js [hunk 2]
```diff
@@ -21,6 +24,11 @@ function Header() {
         <div className='searchBar'>
           <input
             onChange={e => setSearchTerm(e.target.value)}
+            onKeyPress={event => {
```
Comment (Karyum) [inline client/src/components/Header/Header.js]: insteaf of 
```javascript
onKeyPress={event => {
                if (event.key === 'Enter') {
                  window.location = handleLink()
                }
              }}
```

change to 
```javascript
onKeyPress={event => event.key === 'Enter' && window.location = handleLink()
```

File: client/src/components/Header/Header.js [hunk 3]
```diff
@@ -35,10 +43,10 @@ function Header() {
         </div>
       </div>
       <div>
-        <Link to={'/'}>
+        <Link to={'/loginPage'}>
```
Comment (Karyum) [inline client/src/components/Header/Header.js]: again no need for the curly braces `<Link to='/loginPage'>` is enought

File: client/src/components/Header/Header.js (hunk 2 above)
Comment (aysam199) [reply client/src/components/Header/Header.js]: Parsing error: Invalid left-hand side in assignment expression
this error

Comment (Karyum) [reply client/src/components/Header/Header.js]: ahh this is a linter problem, yeah for now don't make the change

**Pull request merged**